macrosynergy/panel/panel_calculator.py

```diff
@@ -199,7 +198,7 @@ def panel_calculator(
     )
 
     # E. Create all required wide dataframes with category names.
-
+    df = df.add_ticker_column()
```

Comment:
This will make memory usage larger than it was before since it now stores this column in memory the entire time. 
Not massively against it since it looks cleaner and uses new functionality but thought I'd highlight this as well. And if the ticker column itself is also categorical then it should be more efficient anyway